src/pages/nft/class/_classId/index.vue

```diff
@@ -140,6 +140,14 @@
             </ul>
           </section>
 
+          <NFTBookSignatureBanner
+            v-if="nftHasSignImage"
+            tag="section"
+            :name="iscnWorkAuthor"
```

Comment:
Should we set a fallback with `creatorDisplayName`? i.e. `iscnWorkAuthor || creatorDisplayName`